client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getEditorNewPostPath } from 'state/ui/editor/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+// We discard all other props in connect() to make sure the component only receives step objects.
+const TodoLinks = steps =>
```

Comment:
I'm wondering if this is the best name for this component. Could we rename it to `NextSteps` or if you have a better suggestion?

Reply:
Sounds good. A couple of implications tho:
* We're using `steps__summary-entry todo` as our `className` right now. We should be consistent; change the `todo` to `next`.
* The left column has `steps__summary-entry completed`. If we turn that into a component, it should be called `CompletedSteps`.

I think that should work.

Reply:
Your suggestions sound good to me 👍

Reply:
Oh, actually, the `todo` className is also used in the left hand side column for unfinished tasks.  `todo` makes sense there -- it's about the checkmark vs X styling. So I should maybe leave the className alone after all.